### PR TITLE
ARROW-2560: [Rust] The Rust README should include Rust-specific information on contributing

### PR DESCRIPTION
**Pull request by paddyhoran**

### CODE REVIEW

File: rust/README.md [hunk 1]
```diff
@@ -51,8 +59,24 @@ cargo run --example read_csv
...
+to this location and then `cargo test` as usual.
+
+Our CI uses `rustfmt` to check code formatting.  Although the project is
+built and tested against nightly rust we use the stable version of
+`rustfmt`.  So before submitting a PR be sure to run the following
+and check for lint issues:
+
 ```bash
-cargo test
+cargo +stable fmt --all -- --check
```
Comment (kszucs) [inline rust/README.md]: @paddyhoran How about [creating a pre-commit](https://pre-commit.com/#new-hooks) hook for this?
We have our pre-commit hooks here: https://github.com/apache/arrow/blob/master/.pre-commit-config.yaml

File: rust/README.md [hunk 2]
```diff
@@ -51,8 +59,24 @@ cargo run --example read_csv
...
 
+Parquet support in Arrow requires data to test against, this data is in a
+git submodule.  To pull down this data run the following:
+
+```bash
+git submodule update --init
+```
+
+The data can then be found in `cpp/submodules/parquet_testing/data`.
+Create a new environment variable called `PARQUET_TEST_DATA` to point
```
Comment (kszucs) [inline rust/README.md]: We should set this as the default if `PARQUET_TEST_DATA` is not defined.

**Review by kszucs — APPROVED** [review]
LGTM

File: rust/README.md (hunk 1 above)
Comment (paddyhoran) [reply rust/README.md]: Yep, sounds like a good idea...

File: rust/README.md (hunk 2 above)
Comment (paddyhoran) [reply rust/README.md]: Yep, also a good idea.  Thanks @kszucs

**Pull request closed without merging**